qt/launcher-specific

```diff
@@ -9,6 +9,8 @@
 export QTCHOOSER_NO_GLOBAL_DIR=1
 if [ "$USE_qt5" = true ]; then
   export QT_SELECT=snappy-qt5
+elif [ "$USE_qt5-app-platform" = true ]; then
```

Comment:
Here or above (under that variable condition), I would add a test like:
`ls -d $SNAP/ubuntu-app-runtime >/dev/null 2>&1` and it that fails, prints:
```
You need to connect the ubuntu-app-runtime package with your application to reuse shared assets, please run:
snap install ubuntu-app-runtime
snap connect ubuntu-app-runtime:platform $SNAP_NAME:platform
```
For instance :)